<launch>

  <rosparam command="load" file="$(find dynamixel_cpp)/config/dynamixel_head.yaml"/>

  <node name="dynamixel_node" type="dynamixel_node" pkg="dynamixel_cpp" output="screen"/>

</launch>
